2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,74 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let honeyCake =
+{
+  tittle: "Honey cake",
+  servings: 8,
+  ingredients:["honey", "sugar", "eggs", "flour", "sour cream"]
+}
+console.log(honeyCake.tittle);
+console.log(`Serves: ${honeyCake.servings}`);
+console.log(`Ingredients:`);
+
+for (let i = 0; i<honeyCake.ingredients; i++){
+  console.log(honeyCake.ingredients[i])
+}
+
+let lasagne =
+{
+  tittle: "Lasagne",
+  servings: 6,
+  ingredients:["mince beef", "lasagne sheets", "tomatoes", "onion", "sauce"]
+}
+console.log(lasagne.tittle);
+console.log(`Serves: ${lasagne.servings}`);
+console.log(`Ingredients:`);
+
+for (let i = 0; i<lasagne.ingredients; i++){
+  console.log(lasagne.ingredients[i])
```

Comment:
here we need '.length' or else the function will not know how many times to loop around. Instead of doing iterations is there another way you are able to loop in javaScript? have a look a the forEach method.